### PR TITLE
EGW doc changes

### PR DESCRIPTION
**Pull request by sridhartigera**
For CE 3.16/next
For CC 3.16

Remove 
1. Enable Policy sync API
2. Copy pull secret 

as these 2 are done now by the operator.

### CODE REVIEW

**Review by fasaxc — DISMISSED** [review]
LGTM, always nice to remove manual steps from the docs :-)

File: calico-enterprise/networking/egress/egress-gateway-aws.mdx [hunk 1]
```diff
@@ -405,19 +403,9 @@ kubectl patch felixconfiguration default --type='merge' -p \
...
-
-```batch
-kubectl patch felixconfiguration.p default --type='merge' -p \
-    '{"spec":{"policySyncPathPrefix":"/var/run/nodeagent"}}'
-```
-
 :::note
 
-- `egressIPSupport` and `policySyncPathPrefix` must be the same on all cluster nodes, so you should only set them in the
+- `egressIPSupport` must be the same on all cluster nodes, so you should only set them in the
```
Comment (bmckercher123) [inline calico-enterprise/networking/egress/egress-gateway-aws.mdx]:
```suggestion
- `egressIPSupport` must be the same on all cluster nodes, so you should set them only in the `default` FelixConfiguration resource.
```

File: calico-enterprise/networking/egress/egress-gateway-azure.mdx [hunk 2]
```diff
@@ -349,35 +349,12 @@ kubectl patch felixconfiguration default --type='merge' -p \
...
-
-```batch
-kubectl patch felixconfiguration.p default --type='merge' -p \
-    '{"spec":{"policySyncPathPrefix":"/var/run/nodeagent"}}'
-```
-
 :::note
 
-`egressIPSupport` and `policySyncPathPrefix` must be the same on all cluster nodes, so you should only set them in the `default` FelixConfiguration resource.
+`egressIPSupport` must be the same on all cluster nodes, so you should only set them in the `default` FelixConfiguration resource.
```
Comment (bmckercher123) [inline calico-enterprise/networking/egress/egress-gateway-azure.mdx]:
```suggestion
`egressIPSupport` must be the same on all cluster nodes, so you should set them only in the `default` FelixConfiguration resource.
```

**Review by bmckercher123 — APPROVED** [review]
LGTM.

**Review by bmckercher123 — CHANGES_REQUESTED** [review]
Remove Enable policy policy and Copy pull secret links from the "how to" section.

**Pull request merged**